src/wasm/wat-parser-internal.h

```diff
@@ -405,8 +471,16 @@ struct IntTok {
   }
 };
 
+struct IdTok {
```

Comment:
`IdTok` doesn't have the actual ID string attached. Then why do we need `n` for `IntTok`?

Reply:
`IdTok` doesn't need to perform any transformations on the span stored in the parent `Token`, so it doesn't need to carry any data. In contrast, it would be unfortunate to have to re-parse an integer token every time we wanted its value.